### PR TITLE
PBI 2 - User can login

### PR DESCRIPTION
**Pull request by nofamex**

### CODE REVIEW

File: src/resources/app/pages/login.vue [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+          <v-btn
+            class="mb-4"
+            @click="login"
+            :disabled="!valid"
+            width="100%"
+            color="primary"
+            depressed
+            id="btn-login"
+          >
+            Login
```
Comment (Zsaschz) [inline src/resources/app/pages/login.vue:40]: you can use a loading icon while running the request for better user experience

**Review by Zsaschz — APPROVED** [review]
still got some todos, but i think this pr has covered the basic functionality and can be merged

File: src/resources/api/auth.js [hunk 2]
```diff
@@ -1,23 +1,16 @@
 import api from "./config";
 
-let apiPrefix = "/badaso-api";
+let apiPrefix = "/badaso-api/module/lms/v1/auth/";
```
Comment (AldiNFitrah) [inline src/resources/api/auth.js:3]: please use const since this won't be changed

**Review by FXDROS — APPROVED** [review]
everything seems pretty good

**Review by IlmaMannix — APPROVED** [review]

File: src/resources/api/auth.js [hunk 3]
```diff
@@ -1,23 +1,16 @@
...
 
 export default {
   register(data) {
-    let ep = apiPrefix + "/module/lms/v1/auth/register";
+    let ep = apiPrefix + "register";
```
Comment (AldiNFitrah) [inline src/resources/api/auth.js:7]: woops didn't notice this one before. `ep` is not clear enough IMO, please change it to something like `url`

File: src/resources/api/auth.js [hunk 4]
```diff
@@ -1,23 +1,16 @@
...
-        let timeNow = date.getTime();
-        localStorage.setItem(
-          window.btoa("tokenAccessTime"),
-          window.btoa(timeNow)
-        );
-      }
-    });
+    return response;
+  },
+  async login(data) {
```
Comment (AldiNFitrah) [inline src/resources/api/auth.js:11]: whitespace please

File: src/resources/api/auth.js [hunk 5]
```diff
@@ -1,23 +1,16 @@
...
+    let ep = apiPrefix + "login";
```
Comment (AldiNFitrah) [inline src/resources/api/auth.js:12]: same for this one, `ep` is not descriptive

File: src/resources/app/pages/login.vue [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+          </v-btn>
+
+          <v-btn
+            @click="submit"
```
Comment (AldiNFitrah) [inline src/resources/app/pages/login.vue:44]: where is this `"submit"` referring to?

**Review by AldiNFitrah — CHANGES_REQUESTED** [review]

File: src/resources/app/pages/login.vue (hunk 6 above)
Comment (nofamex) [reply src/resources/app/pages/login.vue:44]: submit function

**Pull request merged**